stonesoup/feeder/filter.py

```diff
@@ -91,25 +107,25 @@ def detections_gen(self):
             yield time, filtered_detections
 
 
-class BoundingBoxDetectionReducer(Feeder):
-    """ Reduce detections by selecting only ones placed within the limits of a
-        n-dimensional bounding box, defined on the detection coordinate space.
+class BoundingBoxDetectionReducer(DetectionFeeder, GroundTruthFeeder):
```

Comment:
Would make sense to rename this to ``BoundingBoxDataReducer`` or just ``BoundingBoxReducer``.